client/landing/gutenboarding/get-available-designs.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Internal dependencies
+ */
+const availableDesignsConfig = require( './available-designs-config.json' );
+
+/**
+ * This function reads the available designs from the config. This is written
+ * in raw JS so that the node script for generating previews can use this to
```

Comment:
```suggestion
 * in CommonJS so that the node script for generating previews can use this to
```

Nitpick, but I find it a bit clearer than 'raw JS' :grimacing:

Reply:
you are very right